composer.json

```diff
@@ -41,7 +41,7 @@
         "ext-dom": "*",
         "ext-libxml": "*",
         "sabberworm/php-css-parser": "^8.7.0",
-        "symfony/css-selector": "^5.4.0 || ^6.0.0 || ^7.0.0"
+        "symfony/css-selector": "^5.4.35 || ~6.3.12 || ~6.4.3 || ^7.0.3"
```

Comment:
["~1.2 is equivalent to >=1.2 <2.0.0, while ~1.2.3 is equivalent to >=1.2.3 <1.3.0."](https://getcomposer.org/doc/articles/versions.md#tilde-version-range-).  So this would not permit version 6.5.

If Symfony adheres to semver, we could just require `^6.4.3` and not even allow `~6.3.12`, since there should be no breaking changes between 6.3 and 6.4.  But I think it's still better to also allow 6.3 (since we have no reason not to), just in case there is an edge case where a (misconfigured) package permits 6.3 but not 6.4.

Reply:
```suggestion
        "symfony/css-selector": "^5.4.35 || ~6.3.12 || ^6.4.3 || ^7.0.3"
```